src/SaveLoad.elm

```diff
@@ -380,15 +380,15 @@ subscriptions model =
         , Ports.logoutComplete (\_ -> GetLoginStatus)
         , Modal.subscriptions model.loadDialogModal ModalAnimation
         ]
-            ++ (case model.machineData of
-                    MachineCreated ->
+            ++ (case ( model.machineData, model.loginState ) of
+                    ( MachineCreated, LoggedIn _ _ ) ->
                         if model.unsavedChanges then
                             [ Time.every 5000 (MachineCreatedMsg << AutoSave) ]
 
                         else
                             []
 
-                    MachineNotCreated ->
+                    ( MachineNotCreated, _ ) ->
```

Comment:
This is causing an error (incomplete cases). Changing this to `(_, _)` fixes the issue for me

Reply:
Yeah, my bad! I realized this and fixed in the newest commit. Also, added a pen icon to make it clear you can edit the name of the machine.

Reply:
This why we have CI! Apparently Travis is making it more difficult for open source projects to get free builds, so we might have to switch....

Reply:
Good to know; we can look at alternatives